### PR TITLE
Update verbiage on user billing

### PR DESCRIPTION
**Pull request by khorne3**

### CODE REVIEW

File: articles/policies/billing.md [hunk 1]
```diff
@@ -88,15 +88,15 @@ Also, more than often the email which receives the receipt is the one that creat
 
 ## In our pricing, what is the difference between internal and external users? Are they different technically?
 
-An active user is a user that has authenticated with username/password, a passwordless connection or any social provider in the last 30 days, counted per application.
+An active user is a user that has authenticated with a username/password combination, a Passwordless connection, or any social provider in the last 30 days.
```
Comment (solepano) [inline articles/policies/billing.md]: should be "per calendar month" instead of "last 30 days"

**Review by solepano — CHANGES_REQUESTED** [review]

File: articles/policies/billing.md [hunk 2]
```diff
@@ -88,15 +88,15 @@ Also, more than often the email which receives the receipt is the one that creat
 
 ## In our pricing, what is the difference between internal and external users? Are they different technically?
 
-An active user is a user that has authenticated with username/password, a passwordless connection or any social provider in the last 30 days, counted per application.
+An active user is a user that has authenticated with a username/password combination, a Passwordless connection, or any social provider in the last calendar month.
```
Comment (solepano) [inline articles/policies/billing.md]: @khorne3 should be *per* calendar month unstead of  *in the last* calendar month. 😬

**Review by solepano — CHANGES_REQUESTED** [review]

File: articles/policies/billing.md (hunk 2 above)
Comment (khorne3) [reply articles/policies/billing.md]: @solepano I don't think *per calendar month* works in this situation:

> An active user is a user that has authenticated with a username/password combination, a Passwordless connection, or any social provider per calendar month.

I think what you're looking for is *in a given calendar month*?

Comment (solepano) [reply articles/policies/billing.md]: 👍 that would work, thanks!

**Comment by khorne3** [comment]
@solepano ~Did you want this shipped this week?~ Please disregard, I just reviewed the Confluence notes and saw 8/13 as merge date. Thanks!

**Review by jeffreylees — APPROVED** [review]

**Review by solepano — APPROVED** [review]

**Pull request merged**